packages/terra-action-header/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-action-header",
+  "main": "lib/ActionHeader.js",
+  "version": "0.0.0",
+  "description": "The terra-action-header component is a header bar that contains sockets for placing actionable items such as buttons and hyperlinks.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-action-header",
+    "ActionHeader",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-base": "^3.1.0",
+    "terra-button": "^2.2.0",
+    "terra-button-group": "^2.2.0",
+    "terra-icon": "^2.2.0",
+    "terra-responsive-element": "^2.2.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "rimraf lib",
+    "compile:build": "babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "test": "npm run test:jest && npm run test:wdio",
+    "test:jest": "jest ./tests/jest/* --config ../../jestconfig.json",
+    "test:wdio": "wdio ../../wdio.conf.js"
+  },
+  "devDependencies": {
```

Comment:
We should be able to drop this dependency. See my comment above.

Reply:
Removed [here](https://github.com/cerner/terra-core/commit/899ef844b8d2ed926f0df690f13e3b97417e8590).